Resources/Views/Submissions/Fields/textarea-input.leaf

```diff
@@ -0,0 +1,20 @@
+<div class="form-group" name="#(key)">
```

Comment:
Should we name this file `textarea` instead? Or `textarea-field`? Thoughts @rasmusebbesen @siemensikkema ?